<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentMetadataDialog</class>
 <widget class="QDialog" name="DocumentMetadataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Document Metadata</string>
  </property>
  <property name="windowIcon">
   <iconset theme="documentinfo">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title</string>
         </property>
         <property name="buddy">
          <cstring>edit_title</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edit_title"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edit_author"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edit_description"/>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="edit_keywords"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Author</string>
         </property>
         <property name="buddy">
          <cstring>edit_author</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="buddy">
          <cstring>edit_description</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Keywords</string>
         </property>
         <property name="buddy">
          <cstring>edit_keywords</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Custom</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="table_widget">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>table_widget</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>DocumentMetadataDialog</receiver>
   <slot>update_item(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DocumentMetadataDialog</receiver>
   <slot>button_clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_item(QTableWidgetItem*)</slot>
  <slot>button_clicked(QAbstractButton*)</slot>
 </slots>
</ui>
